tests/Microsoft.Identity.Test.LabInfrastructure/LabApiConstants.cs

```diff
@@ -3,7 +3,15 @@
 
 namespace Microsoft.Identity.Test.LabInfrastructure
 {
-    public class LabApiConstants
+    public static class LabApiConstants
+    {
+        public const string LabClientId = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
```

Comment:
if we move to use Visual Studio credential for DevBox and UAMI based acccess for CI, we do not need this app flow anymore